test/test_active_support.rb

```diff
@@ -9,6 +9,23 @@ def cache_key
 end
 
 describe 'ActiveSupport::Cache::DalliStore' do
+  # with and without local cache
+  def self.it_wawolc(message, &block)
+    it "#{message} with LocaleCache" do
+      with_cache do
+        @dalli.with_local_cache do
+          instance_eval(&block)
+        end
+      end
+    end
+
+    it "#{message} without LocaleCache" do
+      with_cache do
```

Comment:
Looks like there's a typo in these two messages - 'LocaleCache'.  Plus I'm not really nuts about the method name - very obscure.  Can you come up with something clearer, although potentially more verbose?